tests/integration_tests/functional/test_logging.py

```diff
@@ -243,7 +225,8 @@ def test_api_requests_logs(test_microvm_with_api):
                                 "Status code: 400 Bad Request. "
                                 "Message: {}".format(fault_message))
 
-    assert log_file_contains_strings(microvm.log_data, expected_log_strings)
+    for expected_log_string in expected_log_strings:
```

Comment:
I think it makes more sense to completely remove `expected_log_strings` and call `check_log_message` after every request and only for the specific message that is expected from that request.